js/plugins/pluginPrototype.js

```diff
@@ -0,0 +1,17 @@
+var clinicalTimelinePlugin = function (id, name, spec) {
+	this.id = id;
+	this.name = name;
```

Comment:
One thing that's odd about requiring one to set `id` and `name` for a plugin, is that those variables are only used in `configCheckManager` as far as I can tell. Makes more sense to me to pass that as configuration to `configCheckManager` instead.